Documentation/Installation.md

```diff
@@ -0,0 +1,43 @@
+
+# Installation
+
+<br>
+
+## Dependencies
+
+You will require `prettier`.
+
+<br>
+
+### PNPM
+
+If you use this method, simply add the <br>
+following code to your `.npmrc` config.
+
+```npmrc
+public-hoist-pattern[]=*
+```
+
+<br>
+
+### Manual
+
+Alternatively you can also install `prettier` <br>
+in your `devDependencies` by yourself.
+
+*If you are using `npm`, the prettier* <br>
+*dependency is hoisted automatically.*
```

Comment:
Do not break the sentence. It should have a line length of 120

Reply:
Why would you enforce 120 when 60 is the human readable limit?

Reply:
For me, 120 is more readable and easier to work with than <80. Also, almost all the reading environments have soft wrapping enabled. You can enforce your desired soft wrap in your reading environment (instead of the writing source).